runtime/moonbase/tests/integration_test.rs

```diff
@@ -31,6 +35,58 @@ use parachain_staking::{Bond, NominatorAdded};
 use sp_core::{Public, H160, U256};
 use sp_runtime::DispatchError;
 
+#[test]
+fn verify_pallet_index() {
+	assert_eq!(System::index(), 0);
+	assert_eq!(moonbase_runtime::Utility::index(), 1);
+	assert_eq!(moonbase_runtime::Timestamp::index(), 2);
+	assert_eq!(Balances::index(), 3);
+	assert_eq!(moonbase_runtime::Sudo::index(), 4);
+	assert_eq!(moonbase_runtime::RandomnessCollectiveFlip::index(), 5);
+	assert_eq!(moonbase_runtime::ParachainSystem::index(), 6);
+	assert_eq!(
+		<moonbase_runtime::Runtime as frame_system::Config>::PalletInfo::index::<
+			moonbase_runtime::TransactionPayment,
+		>(),
+		Some(7)
+	);
```

Comment:
Why do you have to call it as a static method sometimes?

Reply:
For some of the pallet instances, when I tried to use the `PalletInfoAccess` trait, it said it wasn't implemented.

```
error[E0599]: no function or associated item named `index` found for struct `pallet_transaction_payment::Module<moonshadow_runtime::Runtime>` in the current scope
  --> runtime/moonshadow/tests/integration_test.rs:49:55
   |
49 |             assert_eq!(moonshadow_runtime::TransactionPayment::index(), 7);
   |                                                                ^^^^^ function or associated item not found in `pallet_transaction_payment::Module<moonshadow_runtime::Runtime>`
error[E0277]: the trait bound `pallet_transaction_payment::Module<moonshadow_runtime::Runtime>: PalletInfoAccess` is not satisfied
  --> runtime/moonshadow/tests/integration_test.rs:49:15
   |
49 |             assert_eq!(<moonshadow_runtime::TransactionPayment as PalletInfoAccess>::index(), 7);
   |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `PalletInfoAccess` is not implemented for `pallet_transaction_payment::Module<moonshadow_runtime::Runtime>`
   |
   = note: required by `frame_support::traits::PalletInfoAccess::index`

error: aborting due to previous error
```

I don't know why this is is the case, but I found the above syntax always works. So, if it looks better, I could change all of them to use the `PalletInfo` trait as the code shown above does instead of `PalletInfoAccess`. What do you think I should do?

Reply:
Consistency is better so I will change all of them to use this syntax.

Reply:
Yeah, that's very weird. The way you have it with the helper function looks nice.